### PR TITLE
[ViewingRoom] Stub out viewing room

### PR DESCRIPTION
**Pull request by damassi**
Force PR: https://github.com/artsy/force/pull/5524

This builds out the static desktop layout for viewing rooms and stubs stuff for the next few phases.

viewing-room

Next:
- Make responsive
- Break components out into actual structure / cleanup
- Wire up to Relay
- Write tests

### CODE REVIEW

File: src/Apps/ViewingRoom/ViewingRoomApp.tsx [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+    setTab("works")
+    scrollToTabBar()
+  }
+
+  return (
+    <AppContainer maxWidth="100%">
+      {/* Top Header */}
+      <Flex
+        height="calc(100vh - 60px)"
+        style={{ borderBottom: `1px solid ${color("black10")}` }}
```
Comment (damassi) [inline src/Apps/ViewingRoom/ViewingRoomApp.tsx:33]: all of this is gonna be cleaned up in subsequent PRs

File: src/Apps/ViewingRoom/routes.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import loadable from "@loadable/component"
+import { graphql } from "react-relay"
+
+const ViewingRoomApp = loadable(() => import("./ViewingRoomApp"))
+
+export const routes = [
+  {
+    path: "/viewing-room",
```
Comment (sweir27) [inline src/Apps/ViewingRoom/routes.tsx:8]: [note for later] We'll eventually want a `/viewing-room/:id/works` route too.

File: src/Apps/ViewingRoom/ViewingRoomApp.tsx [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+  Image,
+  Button,
+  ChevronIcon,
+  ProgressBar,
+} from "@artsy/palette"
+
+const ViewingRoomApp: React.FC = props => {
+  const [selectedTab, setTab] = useState<"statement" | "works">("statement")
+
+  const showWorks = () => {
```
Comment (sweir27) [inline src/Apps/ViewingRoom/ViewingRoomApp.tsx:23]: [Again, just noting for later] but I think we'll want clicking "works" to take you to a new URL.

**Comment by artsyit** [comment]
:rocket: PR was released in [`v26.31.7`](https://github.com/artsy/reaction/releases/tag/v26.31.7) :rocket:

File: src/Apps/ViewingRoom/routes.tsx (hunk 2 above)
Comment (damassi) [reply src/Apps/ViewingRoom/routes.tsx:8]: Ah, nice catch, will update 👍

**Pull request merged**